AK/Format.h

```diff
@@ -432,10 +432,10 @@ void dmesgln(CheckedFormatString<Parameters...>&& fmt, const Parameters&... para
 #endif
 
 template<typename T, typename = void>
-inline constexpr bool HasFormatter = true;
+constexpr bool HasFormatter = true;
```

Comment:
This is not correct, constexpr global variables are not inline - only those with static duration are.

Reply:
I'm surprised this doesn't make the linker cry about multiple definitions